internal/runnerinstall/nomad.go

```diff
@@ -249,6 +249,18 @@ func (i *NomadRunnerInstaller) InstallFlags(set *flag.Set) {
 		Default: defaultRunnerImage,
 	})
 
+	set.StringVar(&flag.StringVar{
+		Name:   "nomad-runner-cpu",
+		Target: &i.Config.RunnerResourcesCPU,
+		Usage:  "CPU required to run the runner task in MHz.",
+	})
+
+	set.StringVar(&flag.StringVar{
+		Name:   "nomad-runner-memory",
+		Target: &i.Config.RunnerResourcesMemory,
```

Comment:
Weird, did we accidentally delete these install flags or something? Looks like the target var already existed.